backend/.env.example

```
@@ -0,0 +1,18 @@
+# NestJS
+NODE_ENV=development
+PORT=3000
+
+# PostgreSQL
+DB_HOST=localhost
+DB_PORT=5432
+DB_USERNAME=postgres
+DB_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)